Misc/NEWS.d/next/Documentation/2021-05-26-18-29-48.bpo-39798.D4UZnR.rst

```diff
@@ -0,0 +1 @@
+updated and corrected readme with some grammatical mistakes
```

Comment:
This entry is definitely not needed, but NEWS entries should also have proper grammar :)

Reply:
I removed it.